pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -381,7 +381,7 @@ func validateCloneVolumeSnapshotSupportVMSnapshotContent(snapshotContents *snaps
 	if backendstorage.IsBackendStorageNeededForVMI(&vm.Spec.Template.Spec) {
 		result = append(result, metav1.StatusCause{
 			Type:    metav1.CauseTypeFieldValueInvalid,
-			Message: "Source Virtual Machine requires backend storage, operation not supported",
+			Message: "Source Virtual Machine uses backend storage (persistent TPM or EFI), operation not supported",
```

Comment:
I think message should be similar to what we talked about in restore: 
"Restore to a different VM is not supported when snapshotted VM has backend storage (persistent TPM or EFI)"
so
"Clone is not supported when source VM has backend storage (persistent TPM or EFI)"